website/templates/website/documentation_page.html

```diff
@@ -1,6 +1,15 @@
 {% extends "website/base_template.html" %}
 {% load customfilters %}
 
+{% block extra_css %}
+<style type="text/css">
+	.section {
+    padding-top: 50px;
```

Comment:
I Guess this is the fix for the permalinks? IF so, does this scale correctly for various screen sizes?